pkg/api/frontendsettings.go

```diff
@@ -24,6 +28,61 @@ import (
 	"github.com/grafana/grafana/pkg/util"
 )
 
+// Returns a file that is easy to check for changes
+// Any changes to the file means we should refresh the frontend
+func (hs *HTTPServer) GetFrontendAssets(c *contextmodel.ReqContext) {
+	hash := sha256.New()
+	keys := map[string]any{}
+
+	// BuildVersion
+	hash.Reset()
+	_, _ = hash.Write([]byte(setting.BuildVersion))
+	_, _ = hash.Write([]byte(setting.BuildCommit))
+	_, _ = hash.Write([]byte(fmt.Sprintf("%d", setting.BuildStamp)))
+	keys["version"] = fmt.Sprintf("%x", hash.Sum(nil))
+
+	// Plugin configs
+	plugins := []string{}
+	for _, p := range hs.pluginStore.Plugins(c.Req.Context()) {
+		plugins = append(plugins, fmt.Sprintf("%s@%s", p.Name, p.Info.Version))
```

Comment:
Should we use `pluginId` instead of `name` here? Also, are we just including plugins here for completeness or is there currently an issue with stale plugin state?

Reply:
good call